pkg/ctl/cmdutils/configfile.go

```diff
@@ -579,7 +577,7 @@ func NewUtilsEnableEndpointAccessLoader(cmd *Cmd, privateAccess, publicAccess bo
 	}
 	l.validateWithConfigFile = func() error {
 		if l.ClusterConfig.VPC == nil {
-			l.ClusterConfig.VPC = api.NewClusterVPC()
+			l.ClusterConfig.VPC = api.NewClusterVPC(false)
```

Comment:
This couldn't be called on an IPv6 cluster? Why not do the same check here too? 🤔

Reply:
huh? Could you expand :smile:

Reply:
In the above this setting is dependent on the ipv6 flag, here it's just false. What's the difference?

Reply:
this func isn't called for `createCluster`, so its not related